api/mraa/led.h

```diff
@@ -46,16 +46,24 @@ extern "C" {
  */
 typedef struct _led* mraa_led_context;
 
+/**
+ * Initialise led_context, based on led index.
+ *
+ *  @param led ID of the LED
+ *  @returns LED context or NULL
+ */
+mraa_led_context mraa_led_init(int led);
```

Comment:
`unsigned int` would be safer choice for the parameter type, alternatively can be guarded by the runtime checks if you have any objections for unsigned.

Reply:
I'd be tempted to go for `int` and runtime check since passing the negative value would result in huge numbers and IMO it's good to report the proper error here.

Reply:
Sure. I see you've added checks - ok with me.